.github/workflows/run-kata-coco-tests.yaml

```diff
@@ -33,6 +33,7 @@ jobs:
           - nydus
         pull-type:
           - guest-pull
+          - host-share-image-block
```

Comment:
Hi @ChengyuZhu6 ,

On the CoCo meeting today it was raised (again) the concerns with the guest-pull running on very limited scenario (and being unrealistically on production). @danmihai1 will be looking into the solution they provided to AKS.

Then there is this PR to enable testing of host-share within TEE (bare-metal) environments; which will require the commit  281ec8d5d859fef493fe2c2e43d893209feef1e1 to clean up containerd's image cache when switching between snapshotters  (a problem on CI on bare-metal only). So I'm going to suggest to enable the host-share tests be running on non-TEE (based on #9424 ) first, then drop 281ec8d5d859fef493fe2c2e43d893209feef1e1 from this PR. This way we avoid mixing up two problems that are already hard to solve in isolation :)

Or, alternatively, de-prioritize host-share for the time being...not sure it is the best course of action.

Cc @fidencio @stevenhorsman